docs/app_configuration_model.md

```diff
@@ -0,0 +1,62 @@
+component = workload + accessories
+From the perspective of the division of labor and collaboration between different roles, platform R&D and SRE are responsible for defining out-of-the-box components and continuously adding new accessory implementations. Application development and SRE use corresponding components/accessories to describe application operation and maintenance intentions, so as to achieve separation of concerns, so that different roles can be more focused and professional to do their jobs well.
+
+#### Pipeline
+
+Tools and platforms provide a default application delivery process, which can meet most application scenarios. For scenarios that require custom application delivery processes, workflow provides certain scalability. A typical delivery workflow consists of multiple steps, and each step corresponds to the business logic that needs to be executed. Typical workflow steps include manual review, data transfer, multi-cluster release, notification, etc. Specifically at the implementation level, the specific execution of each workflow step is the responsibility of the corresponding plug-in, and platform R&D and SRE can implement new plug-ins.
+
+#### Topology
+
+In order to meet the requirements of high availability and disaster recovery, applications running in the actual production environment usually need to deploy multiple computer rooms and multi-clusters. The deployment topology describes the target computer room or cluster for application delivery and the number of copies in the corresponding environment.
+
+#### PolicySets
+
+The policy set is responsible for defining the technical risks and security compliance policies that need to be followed during the specified application delivery process, such as release policies, risk defense policies, and self-healing policies. Based on standard policy development specifications, platform development, infrastructure SRE, and security students can develop new policy sets, and application development and SRE can bind corresponding policy capabilities based on actual application delivery requirements.
+
+#### Dependency
+
+In a complex business system in production practice, there are often intricate dependencies between applications. Dependency is responsible for defining the dependencies between multiple components within an application and cross-applications.
+
```

Comment:
Perhaps we can add a paragraph to make a comparison with OAM since it is a well-known app configuration in the cloud native community.